index.html

```diff
@@ -6,148 +6,184 @@
+    integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+    crossorigin="anonymous"></script>
 </head>
+
 <body class="container-fluid body_section">
-        <header>
-              <nav class="navbar navbar-expand-lg navbar-light ">
-                <a class="navbar-brand brand-name" href="#">Ajit Verma</a>
-                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-                  <span class="navbar-toggler-icon"></span>
-                </button>
-        
-                <div class="collapse navbar-collapse" id="navbarSupportedContent">
-                  <ul class="navbar-nav ml-auto">
-                    <li class="nav-item">
-                      <a class="nav-link headerachor" href="index.html">Home</a>
-                    </li>
-                    <li class="nav-item">
-                      <a class="nav-link headerachor" href="Education.html">Education</a>
-                    </li>
-                    <li class="nav-item">
-                      <a class="nav-link headerachor" href="skill.html">Skills</a>
-                    </li>
-                    <li class="nav-item">
-                      <a class="nav-link headerachor" href="Project.html">Projects</a>
-                    </li>
-                    <li class="nav-item">
-                      <a class="nav-link headerachor" href="contact.html">Contact Me</a>
-                    </li>
-                  </ul>
-              </nav>
-          </header>
+  <header>
+    <nav class="navbar navbar-expand-lg navbar-light navbar-light-mode">
+      <a class="navbar-brand brand-name glow-effect" href="#" onclick="toggleDarkMode()">Ajit Verma</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+        aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <div class="collapse navbar-collapse navbar-collapse-light-mode" id="navbarSupportedContent">
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item">
+            <a class="nav-link headerachor" href="index.html">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link headerachor" href="Education.html">Education</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link headerachor" href="skill.html">Skills</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link headerachor" href="Project.html">Projects</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link headerachor" href="contact.html">Contact Me</a>
+          </li>
+        </ul>
+    </nav>
+  </header>
+
+  <section id="Home">

```

Comment:
Thanks @Arshdeep-13 For adding these features.
Dark mode is breaking in Home page
![image](https://github.com/AjitVerma15/Interactive-Portfolio/assets/53833570/dfe64913-7e10-4918-962d-7ad8df359f07)